src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp

```diff
@@ -159,6 +159,28 @@ float OPCache::get_size_of_cached_ops() {
     return size;
 }
 
+std::string getOpVersion(const ngraph::NodeTypeInfo& type_info) {
```

Comment:
Lets move it to funcTestUtils
The lib is linked to subgraphDumper too. The other place where it used is op_summary

Reply:
ok